app/services/client/sharePopup.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const popup = require('./popup');
+
+class sharePopUp {
+  /**
+ * Create a Popup for share services
+ * @param {Node} shareLink - html anchor tag
+ * @param {string} shareURL - url of page to be shared
+ */
+  constructor(shareLink, shareURL) {
+    this.shareLink = shareLink;
+    this.shareURL = shareURL;
+    this.shareService = this.shareLink.getAttribute('data-shareService') || null;
+    this.shareTitle = this.shareLink.getAttribute('title') || 'Clay Starter';
+
+    this.setDimensions();
+    this.addShareURL();
+    this.addClickHandler();
+  }
+
+  addShareURL() {
+    switch (this.shareService) {
+      case 'twitter':
+        this.shareLink.href = 'https://twitter.com/share?text=' + encodeURIComponent(this.shareTitle) + '&url=' + this.shareURL + '?utm_source=tw&utm_medium=s3&utm_campaign=sharebutton-t';
```

Comment:
What do you think if we use string literal in here?
```suggestion
        this.shareLink.href = `https://twitter.com/share?text=${encodeURIComponent(this.shareTitle)}&url=${this.shareURL}?utm_source=tw&utm_medium=s3&utm_campaign=sharebutton-t`;
```

Reply:
I think it's a good idea.